problems/arrayMethods.js

```diff
@@ -5,7 +5,13 @@
  * @returns {number} Sum of all numbers
  */
 
-const sumArray = () => {};
+const sumArray = (nums) => {
+  let sum = 0;
+  nums.forEach((num) => {
+    return (sum += num);
```

Comment:
no need to return anything inside of the forEach, remember that forEach is used of side affects